nsdb-cluster/src/main/scala/io/radicalbit/nsdb/cluster/NSDbActors.scala

```diff
@@ -32,7 +32,7 @@ import scala.concurrent.ExecutionContextExecutor
 /**
   * Creates the top level actor [[DatabaseActorsGuardian]] and grpc endpoint [[GrpcEndpoint]] based on coordinators
   */
-trait NSDBActors {
+trait NSDbActors {
```

Comment:
it could be maybe called `NsdbActors` to be compliant with all the other names in the project

Reply:
the right nomenclature should be `NSDb`. In the codebase there are many non-compliant.  names.

The only exception is `NsdbCluster.scala` because that name is used by the native packager plugin to generate the sh script used to start the cluster